### PR TITLE
CORDA-351: force update dependencies and suppress vulnerabilities not…

### PR DESCRIPTION
**Pull request by bpaunescu**
Updated some dependencies to newer versions which have solved the vulnerability issues found by the dependency-check analyzer.

### CODE REVIEW

**Comment by fenryka** [comment]
adding @Clintonio to the review for the Gradle stuff, LGTM but I'll give him the final OK

File: .ci/dependency-checker/suppressedLibraries.xml [hunk 1]
```diff
@@ -10,5 +10,22 @@
       <cpe>cpe:/a:apache:struts:2.0.0</cpe>
    </suppress>
     -->
-
+    <suppress>
```
Comment (Clintonio) [inline .ci/dependency-checker/suppressedLibraries.xml]: Can we add a JIRA to re-visit these later? We'll want to enable them if they get fixed.

File: client/jfx/build.gradle [hunk 2]
```diff
@@ -37,6 +37,9 @@ dependencies {
     compile 'org.fxmisc.easybind:easybind:1.0.3'
 
     // Artemis Client: ability to connect to an Artemis broker and control it.
+    // TODO: remove the forced update of commons-collections and beanutils when artemis updates them
+    compile group: 'org.apache.commons', name: 'commons-collections4', version: "$commons_collections_version"
```
Comment (Clintonio) [inline client/jfx/build.gradle]: Please change these to the stringly form. While this form is more type safe it isn't standard across our gradle files.

File: .ci/dependency-checker/suppressedLibraries.xml (hunk 1 above)
Comment (bpaunescu) [reply .ci/dependency-checker/suppressedLibraries.xml]: Added https://r3-cev.atlassian.net/browse/CORDA-745

**Comment by mikehearn** [comment]
I have filed https://issues.apache.org/jira/browse/ARTEMIS-1483 to get Artemis to upgrade Apache CC. They already did upgrade CC once before to fix this issue, however the fix didn't take because they are importing CC via BeanUtils as well. 

Thanks!

**Pull request merged**